data/geany.glade

```diff
@@ -6578,8 +6578,8 @@
                     <property name="can_focus">False</property>
                     <property name="label" translatable="yes">_Edit</property>
                     <property name="use_underline">True</property>
-                    <signal name="select" handler="on_edit1_select" swapped="no"/>
                     <signal name="deselect" handler="on_edit1_deselect" swapped="no"/>
+                    <signal name="select" handler="on_edit1_select" swapped="no"/>
```

Comment:
This has nothing to do with the pull request but I didn't want to hand-edit it out, otherwise it will just show up in some future commit. I edited the file with Glade 3.8.5.

Reply:
Weird, but OK